sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 5.12.0-beta.1 (Unreleased)
+## 6.0.0-beta.1 (Unreleased)
```

Comment:
```suggestion
## 5.12.0-beta.1 (Unreleased)
```
We've been asked to make this a minor because we're not sure whether or not we'll keep this binary breaking change for the next GA version and we can't release a lower major of a GA after a preview.